day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,27 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(toppings)
+    @toppings = toppings
+    p "You added: #{toppings}"
+  end
+
+  def remove_topping(toppings)
+    p "You removed: #{toppings}"
```

Comment:
How would you change it so that you're removing and element from the existing array?